spec/granite/form/model/validations/nested_spec.rb

```diff
@@ -143,6 +143,106 @@
       it do
         expect { subject.valid? }.not_to raise_error
         expect(subject).not_to be_valid
+        expect(subject.errors.count).to eq(1)
+      end
+    end
+  end
+
+  context 'object field is invalid and has representation of that field' do
```

Comment:
This looks very similar to `'object field is invalid and referenced object does not include AutosaveAssociation'`, I think they can be merged and this could be subcontext that adds more things.